src/RelationalModel/Relation.js

```diff
@@ -107,7 +115,7 @@ var Relation = module.exports = kind(
 	/**
 	* @private
 	*/
-	setRelated: function (related) {
+	setRelated: function (related, opts) {
```

Comment:
Curious if this having this unused parameter here was intentional, to match the overriding done by its subkinds.